pkg/apis/kops/networking.go

```diff
@@ -18,18 +18,19 @@ package kops
 
 // NetworkingSpec allows selection and configuration of a networking plugin
 type NetworkingSpec struct {
-	Classic    *ClassicNetworkingSpec    `json:"classic,omitempty"`
-	Kubenet    *KubenetNetworkingSpec    `json:"kubenet,omitempty"`
-	External   *ExternalNetworkingSpec   `json:"external,omitempty"`
-	CNI        *CNINetworkingSpec        `json:"cni,omitempty"`
-	Kopeio     *KopeioNetworkingSpec     `json:"kopeio,omitempty"`
-	Weave      *WeaveNetworkingSpec      `json:"weave,omitempty"`
-	Flannel    *FlannelNetworkingSpec    `json:"flannel,omitempty"`
-	Calico     *CalicoNetworkingSpec     `json:"calico,omitempty"`
-	Canal      *CanalNetworkingSpec      `json:"canal,omitempty"`
-	Kuberouter *KuberouterNetworkingSpec `json:"kuberouter,omitempty"`
-	Romana     *RomanaNetworkingSpec     `json:"romana,omitempty"`
-	AmazonVPC  *AmazonVPCNetworkingSpec  `json:"amazonvpc,omitempty"`
+	Classic         *ClassicNetworkingSpec         `json:"classic,omitempty"`
+	Kubenet         *KubenetNetworkingSpec         `json:"kubenet,omitempty"`
+	External        *ExternalNetworkingSpec        `json:"external,omitempty"`
+	CNI             *CNINetworkingSpec             `json:"cni,omitempty"`
+	Kopeio          *KopeioNetworkingSpec          `json:"kopeio,omitempty"`
+	Weave           *WeaveNetworkingSpec           `json:"weave,omitempty"`
+	Flannel         *FlannelNetworkingSpec         `json:"flannel,omitempty"`
+	Calico          *CalicoNetworkingSpec          `json:"calico,omitempty"`
+	Canal           *CanalNetworkingSpec           `json:"canal,omitempty"`
+	Kuberouter      *KuberouterNetworkingSpec      `json:"kuberouter,omitempty"`
+	Romana          *RomanaNetworkingSpec          `json:"romana,omitempty"`
+	AmazonVPC       *AmazonVPCNetworkingSpec       `json:"amazonvpc,omitempty"`
+	AmazonVPCIPVlan *AmazonVPCIPVlanNetworkingSpec `json:"amazonvpcipvlan,omitempty"`
```

Comment:
The name cni-ipvlan-vpc-k8s doesn't really roll off the tongue so AmazonVPCIPVlan was the best I could come up with